### PR TITLE
Fixes for issues that make Cosmos not compile

### PR DESCRIPTION
**Pull request by VisualDevelopment**
Bumped Inno Setup Version, since the latest one is Inno Setup 6, which makes the builder not recognize the install.

### CODE REVIEW

**Comment by quajak** [comment]
I think it makes sense more sense to check for both inno setup 5 and 6, so we can support both versions.

**Comment by VisualDevelopment** [comment]
Hmmm.... Alright

**Comment by VisualDevelopment** [comment]
This fixes issue #1150

File: source/Cosmos.Build.Builder/Services/InnoSetupService.cs [hunk 1]
```diff
@@ -5,25 +5,25 @@ namespace Cosmos.Build.Builder.Services
...
-                    if (key?.GetValue("InstallLocation") is string innoSetupPath)
-                    {
-                        if (Directory.Exists(innoSetupPath))
-                        {
-                            return innoSetupPath;
-                        }
-                    }
-                }
+				for(int i = 5; i < 7; i++){
+					using (var key = localMachineKey32.OpenSubKey(System.String.Format(@"SOFTWARE\Microsoft\Windows\CurrentVersion\Uninstall\Inno Setup {0}_is1", i), false))
```
Comment (jp2masa) [inline source/Cosmos.Build.Builder/Services/InnoSetupService.cs]: Nit: I think it would be cleaner to update the const string and use it here.

**Comment by VisualDevelopment** [comment]
What do you mean exactly?

**Comment by VisualDevelopment** [comment]
But okay, i guess i will just update the string

**Comment by VisualDevelopment** [comment]
I don't know why all these files are missing, though

**Comment by quajak** [comment]
Could you try to undo all the changes to the files except InnoSetupService.cs, as they are not needed?

**Comment by VisualDevelopment** [comment]
Yeah, sure, new commit coming in a bit.

**Comment by VisualDevelopment** [comment]
I'm having a problem uhh

**Comment by VisualDevelopment** [comment]
@quajak It says the files are ignored by my .gitignore, which i did not change

**Comment by VisualDevelopment** [comment]
I found a fix

**Comment by quajak** [comment]
Nice. Could you please also revert the changes to BaseIOGroups.cs, Bootstrap.cs and VMWareSVGAII.cs?

**Comment by VisualDevelopment** [comment]
There aren't any changes if you look closely, I don't know why github thinks I did changes.

**Comment by quajak** [comment]
I don't know exactly what changed, most likely spaces were converted to tabs. The easiest way to fix these changes is to revert the two commits and only change that one line again.

**Comment by VisualDevelopment** [comment]
It's telling me the files are the same after overwriting them with freshly pulled ones.

**Comment by VisualDevelopment** [comment]
I'll do your solution then.

**Comment by VisualDevelopment** [comment]
Wait what???
Now more files are deleted???
give me a break man xd

**Comment by quajak** [comment]
Maybe its easier, to close this pr and open a new branch with the single line change and make a new pr.

**Comment by VisualDevelopment** [comment]
I guess i'll do that

**Pull request closed without merging**